src/custom/components/AccountDetails/Transaction.tsx

```diff
@@ -488,10 +497,10 @@ export default function Transaction({ hash: id }: { hash: string }) {
             <TransactionStatusText>
               {isCancelling ? (
                 <MouseoverTooltip text={activity.summary || id}>
-                  {getActivitySummary({ activityData, id })}
+                  {ActivitySummary({ activityData, id })}
```

Comment:
Even more component-y:

```suggestion
                  <ActivitySummary activityData={activityData} id={id} />
```